Cannot build test files after switch to `mise` - need to run `mise trust`

**Describe the bug**
After updating to commit fe87aafdb216596dcc00c3f6151119f7c5cfa9bd, I'm seeing the following failure with a clean re-clone of `riscv-arch-tests`:

```
$  CONFIG_FILES="config/sail/sail-rv64-max/test_config.yaml config/sail/sail-rv32-max/test_config.yaml" FAST=True make --jobs
╭────────────────────────────────────────────────────────────────────── Traceback (most recent call last) ───────────────────────────────────────────────────────────────────────╮
│ /home/mundkur/src/hw/sail-ol/tests/sail-riscv-tests/riscv-arch-test/framework/src/act/parse_udb_config.py:40 in _ensure_udb_installed                                          │
│                                                                                                                                                                                │
│    37 │                                                                                                                                                                        │
│    38 │   # Check if all gems (including udb) are installed at the correct versions                                                                                            │
│    39 │   try:                                                                                                                                                                 │
│ ❱  40 │   │   subprocess.run(["bundle", "check"], check=True, cwd=gemfile.parent, capture_outp                                                                                 │
│    41 │   │   return  # All gems satisfied — correct version is installed                                                                                                      │
│    42 │   except FileNotFoundError as e:                                                                                                                                       │
│    43 │   │   raise RuntimeError(                                                                                                                                              │
│                                                                                                                                                                                │
│ /usr/lib/python3.12/subprocess.py:548 in run                                                                                                                                   │
│                                                                                                                                                                                │
│    545 │   │   kwargs['stdout'] = PIPE                                                                                                                                         │
│    546 │   │   kwargs['stderr'] = PIPE                                                                                                                                         │
│    547 │                                                                                                                                                                       │
│ ❱  548 │   with Popen(*popenargs, **kwargs) as process:                                                                                                                        │
│    549 │   │   try:                                                                                                                                                            │
│    550 │   │   │   stdout, stderr = process.communicate(input, timeout=timeout)                                                                                                │
│    551 │   │   except TimeoutExpired as exc:                                                                                                                                   │
│                                                                                                                                                                                │
│ /usr/lib/python3.12/subprocess.py:1026 in __init__                                                                                                                             │
│                                                                                                                                                                                │
│   1023 │   │   │   │   │   self.stderr = io.TextIOWrapper(self.stderr,                                                                                                         │
│   1024 │   │   │   │   │   │   │   encoding=encoding, errors=errors)                                                                                                           │
│   1025 │   │   │                                                                                                                                                               │
│ ❱ 1026 │   │   │   self._execute_child(args, executable, preexec_fn, close_fds,                                                                                                │
│   1027 │   │   │   │   │   │   │   │   pass_fds, cwd, env,                                                                                                                     │
│   1028 │   │   │   │   │   │   │   │   startupinfo, creationflags, shell,                                                                                                      │
│   1029 │   │   │   │   │   │   │   │   p2cread, p2cwrite,                                                                                                                      │
│                                                                                                                                                                                │
│ /usr/lib/python3.12/subprocess.py:1955 in _execute_child                                                                                                                       │
│                                                                                                                                                                                │
│   1952 │   │   │   │   │   if errno_num != 0:                                                                                                                                  │
│   1953 │   │   │   │   │   │   err_msg = os.strerror(errno_num)                                                                                                                │
│   1954 │   │   │   │   │   if err_filename is not None:                                                                                                                        │
│ ❱ 1955 │   │   │   │   │   │   raise child_exception_type(errno_num, err_msg, err_filename)                                                                                    │
│   1956 │   │   │   │   │   else:                                                                                                                                               │
│   1957 │   │   │   │   │   │   raise child_exception_type(errno_num, err_msg)                                                                                                  │
│   1958 │   │   │   │   raise child_exception_type(err_msg)                                                                                                                     │
╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
FileNotFoundError: [Errno 2] No such file or directory: 'bundle'

The above exception was the direct cause of the following exception:

╭────────────────────────────────────────────────────────────────────── Traceback (most recent call last) ───────────────────────────────────────────────────────────────────────╮
│ /home/mundkur/src/hw/sail-ol/tests/sail-riscv-tests/riscv-arch-test/framework/src/act/act.py:110 in run_act                                                                    │
│                                                                                                                                                                                │
│   107 │   │   config_dir.mkdir(parents=True, exist_ok=True)                                                                                                                    │
│   108 │   │                                                                                                                                                                    │
│   109 │   │   # UDB integration                                                                                                                                                │
│ ❱ 110 │   │   generate_udb_files(config.udb_config, config_dir)                                                                                                                │
│   111 │   │   implemented_extensions = get_implemented_extensions(config_dir / "extensions.txt                                                                                 │
│   112 │   │   config_params = get_config_params(config.udb_config)                                                                                                             │
│   113                                                                                                                                                                          │
│                                                                                                                                                                                │
│ /home/mundkur/src/hw/sail-ol/tests/sail-riscv-tests/riscv-arch-test/framework/src/act/parse_udb_config.py:100 in generate_udb_files                                            │
│                                                                                                                                                                                │
│    97 │   │   not (output_dir / "extensions.txt").exists()                                                                                                                     │
│    98 │   │   or (output_dir / "extensions.txt").stat().st_mtime < udb_config_file.stat().st_m                                                                                 │
│    99 │   ):                                                                                                                                                                   │
│ ❱ 100 │   │   _ensure_udb_installed()                                                                                                                                          │
│   101 │   │   validate_udb_config(udb_config_file)                                                                                                                             │
│   102 │   │   generate_extension_list(udb_config_file, output_dir)                                                                                                             │
│   103                                                                                                                                                                          │
│                                                                                                                                                                                │
│ /home/mundkur/src/hw/sail-ol/tests/sail-riscv-tests/riscv-arch-test/framework/src/act/parse_udb_config.py:43 in _ensure_udb_installed                                          │
│                                                                                                                                                                                │
│    40 │   │   subprocess.run(["bundle", "check"], check=True, cwd=gemfile.parent, capture_outp                                                                                 │
│    41 │   │   return  # All gems satisfied — correct version is installed                                                                                                      │
│    42 │   except FileNotFoundError as e:                                                                                                                                       │
│ ❱  43 │   │   raise RuntimeError(                                                                                                                                              │
│    44 │   │   │   "udb command not found and 'bundle' is not available. See the README for ins                                                                                 │
│    45 │   │   ) from e                                                                                                                                                         │
│    46 │   except subprocess.CalledProcessError:                                                                                                                                │
╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
RuntimeError: udb command not found and 'bundle' is not available. See the README for installation instructions.
make: *** [Makefile:113: elfs] Error 1
```


**Additional context**

I've got `ruby` and `mise` installed:
```
$ ruby --version
ruby 3.2.3 (2024-01-18 revision 52bb2ac0a6) [x86_64-linux-gnu]

$ mise doctor
version: 2026.4.11 linux-x64 (2026-04-13)
activated: yes
shims_on_path: no
self_update_available: yes

build_info:
  Target: x86_64-unknown-linux-gnu
  Features: OPENSSL, RUSTLS_NATIVE_ROOTS, SELF_UPDATE
  Built: Mon, 13 Apr 2026 10:52:24 +0000
  Rust Version: rustc 1.94.1 (e408947bf 2026-03-25)
  Profile: release

shell:
  /bin/bash
  GNU bash, version 5.2.21(1)-release (x86_64-pc-linux-gnu)
  Copyright (C) 2022 Free Software Foundation, Inc.
  License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

  This is free software; you are free to change and redistribute it.
  There is NO WARRANTY, to the extent permitted by law.
[...]
```

--- a/README.md
+++ b/README.md
@@ -159,7 +159,7 @@ git clone https://github.com/riscv/riscv-arch-test
 On entering the top-level directory of the repository for the first
 time, or on the first use of one of the build commands below, you may
 see messages or a prompt from `mise` requiring enabling trust before
-the `.mise.toml` configuration file can be used.  This can be done by
+the `.mise.toml` configuration file can be used. This can be done by
 selecting `Yes` in the prompt or with the following command in the
 top-level directory of the repository:
 

--- a/README.md
+++ b/README.md
@@ -70,6 +70,10 @@ mise --version
 >
 > For alternate installation instructions that do not require mise, see Advanced Installation - COMING SOON.
 
+> [!NOTE]
+>
+> See note on enabling trust in the `.mise.toml` file below.
+
 #### 3. RISC-V Compiler (GCC or LLVM)
 
 The ACT framework is compatible with GCC/Binutils or LLVM/Clang. Only the latest release of each is officially supported and tested in CI.
@@ -150,6 +154,18 @@ Clone the `riscv-arch-test` repo:
 
 ```bash
 git clone https://github.com/riscv/riscv-arch-test
+```
+
+On entering the top-level directory of the repository for the first
+time, or on the first use of one of the build commands below, you may
+see messages or a prompt from `mise` requiring enabling trust before
+the `.mise.toml` configuration file can be used.  This can be done by
+selecting `Yes` in the prompt or with the following command in the
+top-level directory of the repository:
+
+```bash
+mise trust .mise.toml
+
 ```
 
 ### Configuration

